tests/test_tsan.c

```diff
@@ -0,0 +1,31 @@
+// Test tsan with config variables
+//
+// REQUIRES: clang
+// RUN: %clang -o %t -fsanitize=thread -g -O1 %s
+// RUN: strings %t 2>&1 | grep -q "tsan"
```

Comment:
It's probably safer (and an indirect test!) to use `llvm-nm %T | grep __tsan` here. This means adding a requirement on `llvm-nm though`.